test/unit/test/pretty_printing_test.cpp

```diff
@@ -37,78 +34,68 @@ auto debug_str = [](auto && v)
     return sstream.str();
 };
 
-TEST(pretty_printing, gtest_output)
+TEST(pretty_printing, char)
 {
-    // warning: these outputs can change due to changes in gtest
     EXPECT_EQ(gtest_str('a'), "'a' (97, 0x61)"s);
     EXPECT_EQ(debug_str('a'), "a"s);
+}
 
+TEST(pretty_printing, tuple)
+{
     EXPECT_EQ(gtest_str(std::make_tuple<int, int>(42, -10)), "(42, -10)"s);
     EXPECT_EQ(debug_str(std::make_tuple<int, int>(42, -10)), "(42,-10)"s);
+}
 
-#if GTEST_INTERNAL_HAS_VARIANT // googletest at head (>=1.10.x)
+TEST(pretty_printing, variant)
+{
     EXPECT_EQ(gtest_str(std::variant<int>{0}), "('int(index = 0)' with value 0)"s);
-#else // googletest <= 1.10.0
-    EXPECT_EQ(gtest_str(std::variant<int>{0}), "0"s);
-#endif
     EXPECT_EQ(debug_str(std::variant<int>{0}), "0"s);
+}
 
-#if GTEST_INTERNAL_HAS_OPTIONAL // googletest at head (>=1.10.x)
+TEST(pretty_printing, optional)
+{
     EXPECT_EQ(gtest_str(std::optional<int>{}), "(nullopt)"s);
-#else // googletest <= 1.10.0
-    EXPECT_EQ(gtest_str(std::optional<int>{}), "<VALUELESS_OPTIONAL>"s);
-#endif
     EXPECT_EQ(debug_str(std::optional<int>{}), "<VALUELESS_OPTIONAL>"s);
+
+    EXPECT_EQ(gtest_str(std::nullopt), "(nullopt)"s);
```

Comment:
```suggestion
```
This is already tested above, no?

Reply:
Where? The other line uses `std::optional<int>{}`

Reply:
true. ops